language/stdlib/modules/LibraVersion.move

```diff
@@ -3,35 +3,44 @@ address 0x1 {
 module LibraVersion {
     use 0x1::CoreAddresses;
     use 0x1::Errors;
-    use 0x1::LibraConfig;
+    use 0x1::LibraConfig::{Self, LibraConfig};
     use 0x1::LibraTimestamp;
+    use 0x1::Roles;
 
     struct LibraVersion {
         major: u64,
     }
 
-    spec module {
-    }
-
     /// Tried to set an invalid major version for the VM. Major versions must be strictly increasing
     const EINVALID_MAJOR_VERSION_NUMBER: u64 = 0;
 
+
+    spec module {
+        /// Verify all functions in this module.
+        pragma verify;
+    }
+
     public fun initialize(
         lr_account: &signer,
     ) {
         LibraTimestamp::assert_genesis();
-        CoreAddresses::assert_libra_root(lr_account);
+        Roles::assert_libra_root(lr_account);
         LibraConfig::publish_new_config<LibraVersion>(
             lr_account,
             LibraVersion { major: 1 },
         );
     }
+    spec fun initialize {
+        include LibraTimestamp::AbortsIfNotGenesis;
+        include Roles::AbortsIfNotLibraRoot{account: lr_account}; // Aborts if not LibraRoot [B19].
```

Comment:
See above, use doc comments if referring to requirements.

Reply:
Done.